docker-compose/bluebutton_server_start.sh

```diff
@@ -13,30 +13,26 @@ echo "SUPER_USER_EMAIL =" ${SUPER_USER_EMAIL}
 echo "SUPER_USER_PASSWORD =" ${SUPER_USER_PASSWORD}
 echo "BB20_ENABLE_REMOTE_DEBUG =" ${BB20_ENABLE_REMOTE_DEBUG}
 echo "BB20_REMOTE_DEBUG_WAIT_ATTACH =" ${BB20_REMOTE_DEBUG_WAIT_ATTACH}
+echo "DJANGO_USER_ID_ITERATIONS=" ${DJANGO_USER_ID_ITERATIONS}
+echo "DJANGO_USER_ID_SALT=" ${DJANGO_USER_ID_SALT}
```

Comment:
James - just noticed these lines which @dtisza1 pointed out on BB2-545. I would assume that these echo'd values will be addressed on both, just pointing out here as well.